src/humanPose/draw.js

```diff
@@ -651,13 +651,21 @@ export class HumanPoseAnalyzer {
     }
 
     /**
-     * Sets all poses in the time interval
+     * Returns a list of poses in the time interval, preferring the historical
+     * data source where available
      * @param {number} firstTimestamp - start of time interval in ms
      * @param {number} secondTimestamp - end of time interval in ms
      * @return {Pose[]} - all poses in the time interval
      */
     getPosesInTimeInterval(firstTimestamp, secondTimestamp) {
-        const poses = this.clones.all.map(clone => clone.pose).filter(pose => pose.timestamp >= firstTimestamp && pose.timestamp <= secondTimestamp);
+        let clonesList = this.clones.all;
+        if (this.clones.historical.length >= this.clones.live.length) {
```

Comment:
A bit confused by this logic, why does the relative clone count matter?

Reply:
I think it would make more sense if we had a way to call this function and specify which list to pull from (or two separate functions). It might cause problems if we ever want to get the live poses but there's existing historical data.

Reply:
Currently, the one purpose this function serves is to feed into a RegionCard which wants the poses on the timeline between the two times. During recording, this is clones.live. After recording but before the live data is cleared (i.e. page reloaded), this is either clones.live or clones.historical, but not both. After the page is reloaded, this is only clones.historical.

I guess my preferred solution is that clones.historical always contains any clones represented on the timeline and I never have to interact with clones.live.

Reply:
Ohhh, I see. I was wondering when clones.live came into the mix and I forgot you have to get the poses from there in the first place during recording. It makes sense to me now, thanks for taking the time to clear that up for me.

Reply:
No problem! The act of explaining this has made me want a better solution so I'm going to take some time to do that